debian/changelog

```diff
@@ -0,0 +1,5 @@
+jellyfin-web (10.6.0-1) unstable; urgency=medium
+
```

Comment:
Is the `/debian` directory required for joining their official repository eventually? I really prefer a generic deployment directory, especially since the web client has managed to keep the root directory clean in comparison to the server.

Reply:
Yes. All our source in in `src/` anyways. I don't think this "cleanliness" of the root should impede proper packaging formats.